04 - Guides, Workflows, & Courses/for Vim users.md

```diff
@@ -0,0 +1,22 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+---
+
+# for Vim users
+
+This site is for all information valuable to users familiar with vim, that want to use Obsidian's vim mode. 
+
+## Basics
+- You can enable Obsidian's vim mode under the settings under `Editor – Advanced – Vim key bindings`.
+- Obsidian uses the [vim emulation from the CodeMirror Editor](https://github.com/codemirror/codemirror5/blob/master/keymap/vim.js), which does not include all vim commands, but most of them.
```

Comment:
I would link to the cm6 vim emulation repo. The `vim.js` deviated from the original cm5 version, and the repo also contains the adapter code.

Reply:
when you can tell me where the CM6 vim.js is? I couldn't find it in any of the CodeMirror repos

Reply:
https://github.com/replit/codemirror-vim